doc/guide_3rd_party.md

```diff
@@ -90,49 +90,50 @@ In order to start the grant consent user flow, redirect the user to the followin
 
 All API endpoints in ETT are versioned using a header based versioning scheme. A request must contain an `X-API-Version` header with a value matching the desired version of the endpoint to use.
 
-### C# example: API request
-
-```csharp
-var token = "<access-token>"; // Access token obtained with client-credentials
-var organizationId = "<organization-id>"; // Id of organization to act on behalf of
-
-using var httpClient = new HttpClient();
-using var request = new HttpRequestMessage(HttpMethod.Get, $"https://demo.energytrackandtrace.dk/wallet-api/wallets?organizationId={organizationId}");
-request.Headers.Authorization = new AuthenticationHeaderValue("bearer", token);
-request.Headers.Add("X-API-Version", "20250101");
-var response = await httpClient.SendAsync(request);
-```
-
 Refer to OpenAPI spec for a description of available API endpoints.
 
-In the example above, an organization id is used to specify which organization to act on behalf of. The available organizations are given by the consents granted to you as a 3rd party client. A list of consenting organizations can be found with a `GET` request to `api/authorization/client/consents`, providing a valid access token that identifies you as a 3rd party client.
+Most API's takes an organization id as query param and is used to specify which organization to act on behalf of. The available organizations are given by the consents granted to you as a 3rd party client. A list of consenting organizations can be found with a `GET` request to `api/authorization/client/consents`, providing a valid access token that identifies you as a 3rd party client.
```

Comment:
Var listen af consenting orgs ikke indbygget i token?

Reply:
Yes. But it's only the ID's. If you want the name of the organization + tin you need to ask for the list.